ios/build.sh

```diff
@@ -30,19 +30,20 @@ fetch ( ) {
 KMEI_RESOURCES=engine/KMEI/KeymanEngine/resources
 KMEI_BUILD_PATH=engine/KMEI/build
 BUNDLE_PATH=$KMEI_RESOURCES/Keyman.bundle/contents/resources
-APP_RESOURCES=keyman/Keyman/Keyman/libKeyman
-APP_BUNDLE_PATH=$APP_RESOURCES/Keyman.bundle
+APP_FRAMEWORK_PATH=keyman/Keyman/KeymanEngine.framework
 APP_BUILD_PATH=keyman/Keyman/build/
 KMW_SOURCE=../web/source
 
+APP_BUNDLE_PATH=$APP_BUILD_PATH/${CONFIG}-iphoneos/Keyman.app
+KEYBOARD_BUNDLE_PATH=$APP_BUILD_PATH/${CONFIG}-iphoneos/SWKeyboard.appex
+ARCHIVE_PATH=$APP_BUILD_PATH/${CONFIG}-iphoneos/Keyman.xcarchive
```

Comment:
- [ ] These three build variables on the `CONFIG` variable are set below at lines 54 and 85; they must come afterward to work properly.